### ISSUE TITLE
Browse mode Find: NVDA reports the instance of the searched string incompletely

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#7547]
# STR #
1. Go to https://github.com/nvaccess/nvda/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%205098 in Mozilla Firefox.
2. Press Ctrl+Home to ensure you are at the top of the webpage.
3. Invoke the NVDA Find dialog by pressing NVDA+Ctrl+F and type ‘5098’ as the string you wish to search. Hit Enter to issue the search.
Note: ‘5098 is present in the first edit field named ‘search this repository’ and the contents of the same are ‘is:issue is:open 5098’.
Expected Result: NVDA should announce the following:
banner landmark 
search landmark 
edit  has auto complete
5098
Actual Result: NVDA announces the following:
banner landmark 
search landmark 
edit  has auto complete
98
# Test Environment #
* NVDA 2017.3
* Firefox 55.0.3
* Windows 8.1 64 bit

**Comment by fisher729** [comment]
Hi.
I can reproduce this. I realise if I type '098' it says '8' instead of '98'

**Comment by Adriani90** [comment]
Note that this issue is still occuring if you are on the first line of the homepage (skip to content). If you press ctrl+hme and then down arrow key, NVDA will then report the complete search string.

**Comment by Adriani90** [comment]
cc: @leonardder, @jcsteh

**Comment by bhavyashah** [comment]
@fisher729  @Adriani90 Do you still experience this issue on the latest Firefox? I no longer actively use Firefox.

**Comment by Adriani90** [comment]
No this is not reproducible anymore in Firefox 117 with NVDA 2023.2. Closing as works for me.

Thanks Bhavya for following up on these issues. Very helpful and appreciated.
